_data/transport.yml

```diff
@@ -149,7 +149,7 @@ websites:
     - name: Norwegian
       url: https://www.norwegian.com/
       twitter: Fly_Norwegian
-      facebook: norwegianairlines
+      facebook: https://www.facebook.com/flynorwegiannow/
```

Comment:
```suggestion
      facebook: flynorwegiannow
```

Reply:
@token91 As you've not set your repository to be changeable by others, I cannot commit Stephen's suggested change, that we need to have this as work out. Will you please commit that suggestion?
// Kai